A94-subchannel-otel-metrics.md

```diff
@@ -72,6 +72,11 @@ grpc.lb.locality        | Optional    | The locality to which the traffic is bei
 grpc.disconnect_error   | Optional    | Reason for disconnection.
 grpc.security_level     | Optional    | Denotes the security level of the connection. Allowed values - "none", "integrity_only" and "privacy_and_integrity".
 
+The resolver attributes for the `grpc.lb.backend_service` and `grpc.lb.locality`
```

Comment:
This is pretty confusing for Java and Go, because resolver attributes aren't passed into subchannels and this doesn't define what will copy them to address attributes. Maybe something closer to:

> The subchannel will be passed attributes with the values for the `grpc.lb.backend_service` and `grpc.lb.locality` labels  (defined in [A89] and [A78] respectively). The subchannel will be recreated when those attributes change. Since only xDS is presently using these labels, the attributes will be set for each endpoint or address by cds (post-A74) or cluster_resolver (pre-A74) LB policies.

Reply:
In Go it uses address attributes, not endpoint attributes. So I was actually serious when saying it will be for each endpoint or address, as it varies per language.